l10n_nl_xaf_auditfile_export/views/templates.xml

```diff
@@ -138,8 +138,9 @@
                 <customerSupplierHistory />
             </customerSupplier>
         </customersSuppliers>
-        <generalLedger>
-            <ledgerAccount t-foreach="self.get_accounts()" t-as="a">
+        <t t-set="accounts" t-value="self.get_accounts()" />
```

Comment:
Should we even allow an export if there are as yet not any accounts for the company?

Reply:
No, but apparently the reason that there were no accounts is that there was another company active in a multi-company setup. So that is still something that needs to be addressed also, maybe with a sudo.